### DISCUSSION TITLE
Annotations spanning multiple lines not working?

### DISCUSSION THREAD

**Discussion by Nate-Wessel** [community/community#122594] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I believe I may have found a bug where annotations on a PR which span multiple lines of code just do not show up. I initially reported it [here](https://github.com/yuzutech/annotations-action/issues/68) assuming it was a problem with this action, but I didn't see any obvious problems after looking at that code, and I didn't find any examples in the github documentation where the `start_line` parameter != the `end_line`. Which makes me wonder if no one is looking out for this case?

Has anyone seen a working example of a multi-line annotation on a GitHub PR out in the wild?

**Comment by filipmilo** [comment]
Were you able to resolve this?

**Reply by Nate-Wessel** [reply]
No, I haven't found a solution for this yet. Some annotations are still not showing up :-/
